src/main/webapp/regexTester.js

```diff
@@ -0,0 +1,148 @@
+//
+// Please maintain tests in config.js.test.html
+// THIS IS A DULICATION OF atlassian-jira-software-cloud-plugin/src/main/resources/com/atlassian/jira/cloud/jenkins/config/JiraCloudPluginConfig/config.js
+// We have moved it here to allow access from the Jelly file
+//
+
```

Comment:
Feel free to ignore this file, its a copy paste of the same file in a different directory, but the only way to access it is to have it in the webapps folder.

it will be the "main" version once we remove the old flow